+nosnoc/+discrete_time_problem/Gcs.m

```diff
@@ -0,0 +1,870 @@
+classdef Gcs < vdx.problems.Mpcc
+    properties
+        model
+        dcs
+        opts
+
+        populated
+    end
+
+    methods
+        function obj = Gcs(dcs, opts)
+            obj = obj@vdx.problems.Mpcc();
+            obj.model = dcs.model;
+            obj.dcs = dcs;
+            obj.opts = opts;
+        end
+
+        function create_variables(obj)
+            dims = obj.dcs.dims;
+            dcs = obj.dcs;
+            model = obj.model;
+            opts = obj.opts;
+
+            % Parameters
+            obj.p.rho_h_p = {{'rho_h_p',1}, 1};
```

Comment:
I guess we dont want a base discrete time problem, where all the parameters e.g., that the same live for sake of readiblity?

Reply:
Yea there is very little overlap. In theory we could move some of  the extremely common code into a base class but the 1% reduction in repetition is probably not worth the readability penalty. I am open to changing this in the future but for now I am happy as it is.

Reply:
I agree.